lib/ansible/utils/module_docs_fragments/default_callback.py

```diff
@@ -16,6 +16,26 @@ class ModuleDocFragment(object):
           - key: display_skipped_hosts
             section: defaults
         type: boolean
+      display_ok_hosts:
+        name: Show 'ok' hosts
+        description: "Toggle to control displaying 'ok' task/host results in a task"
+        default: True
+        env:
+          - name: ANSIBLE_DISPLAY_OK_HOSTS
+        ini:
+          - key: display_ok_hosts
+            section: defaults
+        type: boolean
```

Comment:
missing version_added for options

Reply:
I just copied from the existing sections, which didn't have that key. I've pushed a new commit with that added.

Reply:
normally they don't need it as they got added at same time as the plugin did, but newer options require it.

Reply:
fyi, make it a string "2.7" otherwise it becomes  a float and .. version comparison can break

Reply:
Done